.ci/atime/tests.R

```diff
@@ -119,6 +119,26 @@ test.list <- atime::atime_test_list(
       data.table:::setDT(L)
     },
     Slow = "c4a2085e35689a108d67dacb2f8261e4964d7e12", # Parent of the first commit in the PR that fixes the issue (https://github.com/Rdatatable/data.table/commit/7cc4da4c1c8e568f655ab5167922dcdb75953801)
-    Fast = "1872f473b20fdcddc5c1b35d79fe9229cd9a1d15") # Last commit in the PR that fixes the issue (https://github.com/Rdatatable/data.table/pull/5427/commits)
+    Fast = "1872f473b20fdcddc5c1b35d79fe9229cd9a1d15"), # Last commit in the PR that fixes the issue (https://github.com/Rdatatable/data.table/pull/5427/commits)
+  
+  # Issue reported in: https://github.com/Rdatatable/data.table/issues/3735
+  # To be fixed in: https://github.com/Rdatatable/data.table/pull/4488
+  "Selecting from data.table by row improved in #4488" = atime::atime_test(
```

Comment:
Title seems a bit long, so how about 'Row-wise selection improved in #4488'

Reply:
The name should start with a function, so for [.data.table I suggest using `DT[i]` etc